### PR TITLE
feature:  style page

### PR DESCRIPTION
**Pull request by ngtrongnguyen**

### CODE REVIEW

File: src/stylesheets/component/_index.scss [hunk 1]
```diff
@@ -0,0 +1,2 @@
+@forward 'nav';
+@forward 'mainheader';
```
Comment (Lanng21) [inline src/stylesheets/component/_index.scss:2]: end of line

File: src/stylesheets/component/_mainheader.scss [hunk 2]
```diff
@@ -0,0 +1,63 @@
+@use '../abstract/' as * ;
+.main-header{
+    width: 1043px;
+    height: 880px;
```
Comment (Lanng21) [inline src/stylesheets/component/_mainheader.scss:4]: Don't fix width and height in pixels that will make it difficult for you when you're responsive.

File: src/stylesheets/component/_mainheader.scss [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+/* Brand color/Secondary */
+
+    background: #1E3446;
+}
+.main-texthcontent{
+    position: relative;
+    top: 286px;
+    left: 104px;
+    width: 684px;
+    height: 216px;
```
Comment (Lanng21) [inline src/stylesheets/component/_mainheader.scss:17]: same. And the width and height must not exceed 100px.

File: src/stylesheets/component/_mainheader.scss [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    position: relative;
+    top: 280px;
+    left:104px;
+    
+/* identical to box height, or 150% */
+
+    
+}
+.main-btn:nth-child(1){
+    background-color: #F58634;
```
Comment (Lanng21) [inline src/stylesheets/component/_mainheader.scss:44]: color

File: src/stylesheets/component/_mainheader.scss [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    color: $text-color;
+    border-color: #F58634;
```
Comment (Lanng21) [inline src/stylesheets/component/_mainheader.scss:46]: same

File: src/stylesheets/component/_mainheader.scss [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+    top: 300px;
+    left:104px;
+
+}
+
+// IMG 
+.main-img{
+    position: relative;
+    width: 490px;
+    height: 640px;
```
Comment (Lanng21) [inline src/stylesheets/component/_mainheader.scss:59]: why u fix  width height for img?

File: src/stylesheets/component/_nav.scss [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+
+.header-navbar{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    margin-left: 100px;
+    position: fixed;
+    width: 1250px;
+    height: 106px;
```
Comment (Lanng21) [inline src/stylesheets/component/_nav.scss:11]: same

File: src/stylesheets/component/_nav.scss [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+}
+.nav-logo{
+    position: relative;
+    width: 32px;
+    height: 32px;
+    left: 0px;
+    top: 53px;
+    border: 9px solid;
+    
+    border-color: #F58634  #F58634  #F58634 transparent;
```
Comment (Lanng21) [inline src/stylesheets/component/_nav.scss:50]: color

File: src/stylesheets/component/_nav.scss [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+}
+    
+   
+.nav-login-inside{
+    
+    position: absolute;
+    top: -3px;
+    left: -4px;
+    border: 20px solid;
+    border-color: white #F58634  white white;
```
Comment (Lanng21) [inline src/stylesheets/component/_nav.scss:66]: same

File: src/stylesheets/component/_nav.scss [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+    line-height: 27px;
+    display: flex;
+    align-items: center;
+
+    color: $text-color;
+}
+
+
+.nav-items--regis{
+    color: #F58634;
```
Comment (Lanng21) [inline src/stylesheets/component/_nav.scss:88]: color

File: src/stylesheets/pages/_comment.scss [hunk 11]
```diff
@@ -0,0 +1,18 @@
+@use '../abstract/' as * ;
+.main-comment{
+    position: relative;
+    width: 1442px;
+    height: 508px;
```
Comment (Lanng21) [inline src/stylesheets/pages/_comment.scss:5]: check width height. pls!!

File: src/stylesheets/pages/_comment.scss [hunk 12]
```diff
@@ -0,0 +1,18 @@
...
+    .comment-group{
+
+        .comment-element{
+
+            .comment-elementbg{
+
+            }
+        }
+    }
+}
```
Comment (Lanng21) [inline src/stylesheets/pages/_comment.scss:18]: end of line

File: src/stylesheets/pages/_course.scss [hunk 13]
```diff
@@ -0,0 +1,34 @@
+
+
+.main-course{
+
+    display: block;
+    position: relative;  
+    width: 1233px;
+    height: 554px;
```
Comment (Lanng21) [inline src/stylesheets/pages/_course.scss:8]: same

File: src/stylesheets/pages/_comment.scss (hunk 11 above)
Comment (haunguyen15-tt) [reply src/stylesheets/pages/_comment.scss:5]: Pls do not set a fixed width and height if possible. If u need, use max-width and % to set width and min-height to set height. But I do not recommend a set height

**Comment by haunguyen15-tt** [comment]
I saw u using `px` in all of your code. Pls check this [link](https://www.youtube.com/watch?v=N5wpD9Ov_To&t=183s)

File: src/stylesheets/component/_card-rating.scss [hunk 14]
```diff
@@ -0,0 +1,25 @@
+@use '../abstract/' as *;
+
+:root {
```
Comment (Lanng21) [inline src/stylesheets/component/_card-rating.scss:3]: why u style it here

File: src/stylesheets/component/_nav.scss [hunk 15]
```diff
@@ -0,0 +1,86 @@
...
+    margin: 12px;
+    text-decoration: none;
+}
+.nav-left{
+    display: flex;
+    flex-direction: row;
+    .nav-logo{
+       
+        border: 6px solid;
+        border-color: #F58634  #F58634  #F58634 transparent;
```
Comment (Lanng21) [inline src/stylesheets/component/_nav.scss:46]: color

File: src/stylesheets/pages/_comment.scss [hunk 16]
```diff
@@ -0,0 +1,18 @@
+@use '../abstract/' as * ;
+.main-comment{
+    position: relative;
+    width: 1442px;
+    height: 508px;
+    left: 104px;
+    top: 1400px;
+    background-color: $comment-background;
+    .comment-group{
```
Comment (Lanng21) [inline src/stylesheets/pages/_comment.scss:9]: delete if you don't style

File: src/stylesheets/pages/_course.scss [hunk 17]
```diff
@@ -0,0 +1,44 @@
...
+            border: 0.5px solid #B2B3B3;
+            border-radius: 16px;
+            display: inline-block;
+            margin-top: 10px;
+            box-sizing: content-box;
+            padding: 20px;
+            .card{
+                
+                margin: 10px;
+               .course-item{
```
Comment (Lanng21) [inline src/stylesheets/pages/_course.scss:24]: delete

File: src/stylesheets/pages/_course.scss [hunk 18]
```diff
@@ -0,0 +1,44 @@
...
+
+.main-course{
+
+    display: block;
+    
+    margin: 0;
+    .course-cardtotal{
+        @include flex($direction: row,$justify: space-between, $align: center);
+        gap: 24px;
+
```
Comment (Lanng21) [inline src/stylesheets/pages/_course.scss:11]: break line

File: src/stylesheets/pages/_gallery.scss [hunk 19]
```diff
@@ -0,0 +1,30 @@
+
+.main-gallery{
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0px;
+    gap: 24px;
+
+    position: absolute;
+    width: 1233px;
```
Comment (Lanng21) [inline src/stylesheets/pages/_gallery.scss]: check w h

**Pull request closed without merging**